src/dipali/Bank.java

```diff
@@ -0,0 +1,62 @@
+package dipali;
+
+public class Bank {
+	
+	int debitCnt=0 , creditCnt=0 , printCnt=0;
+	static int debitCount=0 , creditCount=0 , printCount=0;
+	int totalAmount=20000;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value and don't give hardcoded 20000